www/script/project/rtBuilders.js

```diff
@@ -165,19 +129,23 @@ define(function (require) {
                         branch_tags.add(tag.toLowerCase());
                     }
                 });
+
+                if (builderTags.length > 1){
+                    tags.add(builderTags.join(tagSeparator));
```

Comment:
So we'll have the individual tags and then finally an entry we each comma seperated, whats the reasoning behind that need?